services/scanner/health_check_analyzer.go

```diff
@@ -0,0 +1,83 @@
+package scanner
+
+import (
+	"context"
+
+	"github.com/forta-network/forta-core-go/clients/health"
+	"github.com/forta-network/forta-node/clients/messaging"
+	"github.com/forta-network/forta-node/services/components"
+	"github.com/forta-network/forta-node/services/components/botio/botreq"
+	"github.com/forta-network/forta-node/services/components/metrics"
+
+	"github.com/forta-network/forta-core-go/protocol"
+	"github.com/forta-network/forta-node/clients"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// HealthCheckAnalyzerService reads TX info, calls agents, and emits results
+type HealthCheckAnalyzerService struct {
```

Comment:
I couldn't understand the need for this service. Maybe get the count from the health metrics that are already available and use that in the batch? I'm worried that we are doing something over complicated just to get a count and set in the batch.